addons/mail/static/tests/helpers/patch_ui_size.js

```diff
@@ -0,0 +1,111 @@
+/** @odoo-module **/
+
+import { browser } from '@web/core/browser/browser';
+import { MEDIAS, SIZES } from '@web/core/ui/ui_service';
+import { patchWithCleanup } from "@web/../tests/helpers/utils";
+
+/**
+ * Get the min/max width from a MediaQueryList.
+ * eg. (min-width: 250px) and (max-width: 500px) --> [250, 500]
+ *     (min-width: 250px) --> [250, undefined]
+ *
+ * @param {MediaQueryList} mediaQueryList The MediaQueryList to extract the min/max width from.
+ * @returns {number[]} an array containing two elements: min and max width for the given
+ * MediaQueryList.
+ */
+function getMinMaxWidthFromMediaQueryList(mediaQueryList) {
+    const mediaQueryParts = mediaQueryList.media.trim().split('and');
+    let minWidth, maxWidth;
+    if (mediaQueryParts[0].includes('min-width')) {
+        minWidth = parseInt(mediaQueryParts[0].replace(/^\D+|\D+$/g, ''));
+    } else if (mediaQueryParts[0].includes('max-width')) {
+        maxWidth = parseInt(mediaQueryParts[0].replace(/^\D+|\D+$/g, ''));
+    }
+    if (mediaQueryParts.length > 1) {
+        maxWidth = parseInt(mediaQueryParts[1].replace(/^\D+|\D+$/g, ''));
+    }
+    return [minWidth, maxWidth];
+}
```

Comment:
Shouldn't we encode them the other way around so that we don't need to do this? Have the array export, and build the query from the values?